plugins/plugin_compile/project_compile.py

```diff
@@ -895,6 +902,25 @@ def build(self, platform):
                               ( os.path.relpath(cmakefile_dir, build_dir), self._use_sdk ) )
             elif platform == 'mac':
                 self._run_cmd('cmake -GXcode %s' % os.path.relpath(cmakefile_dir, build_dir))
+            elif platform == "win32":
+                ret = utils.get_newest_devenv(self.vs_version)
+                if ret is not None:
+                    ver_num = int(float(ret[2]))
+                    generator = self.CMAKE_VS_GENERATOR_MAP[str(ver_num)]
+                    if generator is not None:
+                        if ver_num >= 16:
+                            # for vs2019 x64 is the default target
+                            self._run_cmd('cmake %s -G "%s" -A win32' % 
+                                (os.path.relpath(cmakefile_dir, build_dir), generator))
+                        else: 
+                            self._run_cmd('cmake %s -G "%s"' % 
```

Comment:
If Visual Studio 2019 is installed in the machine and it's used as the default generator by cmake. It will generate an x64 target, which is not supported by now.